### ISSUE TITLE
Travis targets for python checks

### ISSUE THREAD

**Issue by nilsdeppe** [sxs-collaboration/spectre#1104]
# Feature request:

It would be nice to have checks on Travis for:
- PEP8 formatting in python files
- a linter that performs similar duties to what clang-tidy does for C++ code. E.g. "You should do this instead because it is the modern way of doing things"

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [x] Continuous integration

**Comment by nilsvu** [comment]
Also: Tests in Python 2 and 3 environments

**Comment by nilsdeppe** [comment]
I think `yapf` handles some of this, at least the formatting part. Do we want/need a linter? @nilsleiffischer ?

**Comment by nilsvu** [comment]
Yes, I suggest we add [`pylint`](http://pylint.pycqa.org/en/latest/) to the container and Google's [`.pylintrc`](https://google.github.io/styleguide/pyguide.html#21-lint) to the repo.

**Comment by nilsdeppe** [comment]
I this is actually been done. Closing, can reopen if needed.
